Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Collections/BaseObjectCollection.cs

```diff
@@ -20,6 +20,12 @@ public abstract class BaseObjectCollection : MonoBehaviour
         /// </summary>
         protected List<ObjectCollectionNode> NodeList { get; } = new List<ObjectCollectionNode>();
 
+        /// <summary>
+        /// Read only list of objects in the collection with generated data on the object.
+        /// </summary>
+        public IReadOnlyList<ObjectCollectionNode> CollectionNodes => new List<ObjectCollectionNode>(NodeList) as IReadOnlyList<ObjectCollectionNode>;
```

Comment:
Could you use NodeList instead? This seems like a duplicate. In general, be wary of adding new public members, as it increases API surface we need to support.

Reply:
I suspect this might have been an old merge. I vaguely remember not having access to NodeList at some point. I don't even reference it so I'm not sure why its there. Happy to remove.